app/src/main/java/me/ccrama/redditslide/Activities/Wiki.java

```diff
@@ -83,12 +85,26 @@ private void createCustomCss() {
                 .append("color: ".concat(getHexFromColorInt(ta.getColor(2, Color.BLUE))).concat(";"))
                 .append("; }");
         ta.recycle();
-        // TODO: Implement horizontal scrolling to Wiki table views
-        //customCssBuilder.append("table { display: block; overflow-x: auto; white-space: nowrap; }");
+        customCssBuilder.append("table, code { display: block; overflow-x: scroll; }");
+        customCssBuilder.append("table { white-space: nowrap; }");
```

Comment:
I believe this to have been a poor choice since it causes table rows to extend on single lines infinitely. This should be removed or a max width should be added to the going style